typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -1,6 +1,6 @@
 export enum WarpRouteIds {
   Ancient8EthereumUSDC = 'USDC/ancient8-ethereum',
-  ArbitrumBaseBlastBscEthereumFraxtalLineaModeOptimismZircuitEZETH = 'EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-zircuit',
+  ArbitrumBaseBlastBscEthereumFraxtalLineaModeOptimismSeiTaikoZircuitEZETH = 'EZETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit',
```

Comment:
not sure if you saw Le's recent PR https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4695 that does this too

Reply:
I did this on the plane and hadn't seen his change lol